### PR TITLE
Feature/psp + daofactory tests

### PR DESCRIPTION
**Pull request by novaknole**
## Description

Please include a summary of the change and be sure you follow the contributions rules we do provide [here](./CONTRIBUTIONS.md)

Task: [ID]()

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:

- [ ] I have selected the correct base branch.
- [ ] I have performed a self-review of my own code.
- [ ] I have commented my code, particularly in hard-to-understand areas.
- [ ] I have made corresponding changes to the documentation.
- [ ] My changes generate no new warnings.
- [ ] Any dependent changes have been merged and published in downstream modules.
- [ ] I ran all tests with success and extended them if necessary.
- [ ] I have updated the ``CHANGELOG.md`` file in the root folder.
- [ ] I have tested my code on the test network.

### CODE REVIEW

File: packages/contracts/test/framework/plugin/plugin-setup-processor.ts [hunk 1]
```diff
@@ -625,10 +644,6 @@ describe('Plugin Setup Processor', function () {
           .withArgs(targetDao.address, plugin, preparedSetupId, appliedSetupId);
       });
 
-      it.skip('successfully calls daos multi bulk with correct permissions', async () => {
```
Comment (brickpop) [inline packages/contracts/test/framework/plugin/plugin-setup-processor.ts:628]: No test here?

File: packages/contracts/test/framework/plugin/plugin-setup-processor.ts [hunk 2]
```diff
@@ -1096,10 +1131,6 @@ describe('Plugin Setup Processor', function () {
           .withArgs(preparedSetupId);
       });
 
-      it.skip('successfully calls daos multi bulk with correct permissions', async () => {
```
Comment (brickpop) [inline packages/contracts/test/framework/plugin/plugin-setup-processor.ts:1099]: Not applicable?

File: packages/contracts/test/framework/plugin/plugin-setup-processor.ts [hunk 3]
```diff
@@ -1860,8 +1914,97 @@ describe('Plugin Setup Processor', function () {
...
+          5,
+        ];
+
+        const {plugin, helpers} = await installPlugin(
+          psp,
+          targetDao.address,
+          currentPluginRepoPointer
+        );
+
+        updatePlugin(
```
Comment (brickpop) [inline packages/contracts/test/framework/plugin/plugin-setup-processor.ts:1951]: no await needed?

Comment (brickpop) [reply packages/contracts/test/framework/plugin/plugin-setup-processor.ts]:
```suggestion
        await updatePlugin(
```

**Review by brickpop — DISMISSED** [review]

**Review by mathewmeconry — APPROVED** [review]

**Review by brickpop — APPROVED** [review]

**Review by Rekard0 — APPROVED** [review]
LGTM!

**Pull request merged**